### PR TITLE
chore(docs): documentation

### PR DESCRIPTION
**Pull request by dehamzah**
Documentation for flutter jakarta talks

### CODE REVIEW

**Comment by dehamzah** [comment]
@jfushada can you review my grammar? and please also add additional information if needed.

File: README.md [hunk 1]
```diff
@@ -1 +1,18 @@
-# talks
\ No newline at end of file
+# Flutter Jakarta Talks
+
+We want Flutter Jakarta to be a place for flutter enthusiasts to share knowledge about development with Flutter. This repository is intended as an archive and talks proposal for anyone who wants to be a speaker at our monthly meetup. We usually hold meetups in the last week every 1 month (if possible).
```
Comment (jfushada) [inline README.md]: `We want Flutter Jakarta to be a place for Flutter enthusiasts to share knowledge on Flutter development. This repository is intended as a talk archive and a place to submit talk proposals for anyone who wants to be a speaker at our meetups. We usually hold meetups in the last week of every month (if possible).`

File: README.md [hunk 2]
```diff
@@ -1 +1,18 @@
...
+
+## Submit Your Talks Proposal
+
+Please submit your talk proposal through github issues, you have 30 minutes to bring the talk you want. Make sure your talk proposal includes the following information:
```
Comment (jfushada) [inline README.md]: `Please submit your talk proposal by creating a GitHub Issue. You have 30 minutes to present, and make sure that your talk proposal includes the following information: `

File: README.md [hunk 3]
```diff
@@ -1 +1,18 @@
...
+- Talk Title
+- Target Audience
+- Abstract
+- Outline
+- Speaker Bio
+- Contact
+
+## How Talks Are Selected
+
+There will be 2 speakers in each meetup, we will choose talks based on a theme or similar target audience. We want this community to be a warm place for you for sharing. We will contact you via github or alternative contacts you have informed.
```
Comment (jfushada) [inline README.md]: `There will be 2 speakers in each meetup, and we will chose talks based on a theme or a similar target audience. We want this community to be a warm and comfortable place for you to share. We will contact you via GitHub or alternative information you have provided.`

**Review by jfushada — COMMENTED** [review]
@dehamzah I've already had a look, there's just a couple of changes. Also I was wondering if you were planning to upload the previous 2 meetups' content? 

Also will let you know later if I think of more things to put in here. Thanks!

File: README.md (hunk 3 above)
Comment (dehamzah) [reply README.md]: > @dehamzah I've already had a look, there's just a couple of changes. Also I was wondering if you were planning to upload the previous 2 meetups' content?
> 
> Also will let you know later if I think of more things to put in here. Thanks!

I also planning to upload our previous meetups content here, but i was thinking to separate the repo for talks and workshop, so there will be one more repo like this but for workshops content.

**Pull request merged**